addons/rating/migrations/10.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,7 @@
+---Fields in module 'rating'---
+rating       / rating.rating            / consumed (boolean)            : NEW
```

Comment:
@cubells you have to set this to True if `rating = -1`, which is the previous default value when unset.

Reply:
EDIT: Sorry, the correct comparison value is `!= -1`

Reply:
@pedrobaeza 
done!